scripts/deploy.js

```diff
@@ -0,0 +1,43 @@
+// We require the Hardhat Runtime Environment explicitly here. This is optional
+// but useful for running the script in a standalone fashion through `node <script>`.
+//
+// When running the script with `npx hardhat run <script>` you'll find the Hardhat
+// Runtime Environment's members available in the global scope.
+const hre = require("hardhat");
+const fs = require("fs");
+
+async function main() {
+  // const Greeter = await hre.ethers.getContractFactory("Greeter");
+  // const greeter = await Greeter.deploy("Hello, Hardhat!");
+  const NFTFactory = await hre.ethers.getContractFactory("NFTFactory");
+  const nftFactory = await NFTFactory.deploy("Hovo", "Hakob!");
```

Comment:
optional - we can configure the script to receive constructor params from .env file.